### PR TITLE
Jon/fix/aave/swap-status-display

### PR DESCRIPTION
**Pull request by Jon-edge**
- Update SwapActionOp type
- Update swap description to be more descriptive about the WBTC network being swapped into.
- Fix sprintf call

Screen Shot 2022-09-23 at 12 40 27 PM

#### PR Dependencies

 none

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203020953556924

### CODE REVIEW

File: src/controllers/action-queue/cleaners.ts [hunk 1]
```diff
@@ -112,12 +112,13 @@ const asLoanWithdrawActionOp: Cleaner<LoanWithdrawActionOp> = asObject({
 })
 const asSwapActionOp: Cleaner<SwapActionOp> = asObject({
   type: asValue('swap'),
-  fromWalletId: asString,
-  toWalletId: asString,
+  amountFor: asValue('from', 'to'),
+  walletId: asString,
```
Comment (samholmes) [inline src/controllers/action-queue/cleaners.ts:116]: Why are we adding a third walletId?

File: src/controllers/action-queue/display.ts [hunk 2]
```diff
@@ -100,7 +104,14 @@ async function getActionOpDisplayInfo(account: EdgeAccount, actionOp: ActionOp,
...
         ...baseDisplayInfo,
         title: sprintf(s.strings.action_queue_display_swap_title, fromCurrencyCode, toCurrencyCode),
-        message: sprintf(s.strings.action_queue_display_swap_message, fromCurrencyCode, toCurrencyCode)
+        message: sprintf(
+          s.strings.action_queue_display_swap_message,
+          fromCurrencyCode,
+          config.appName,
+          toCurrencyCode,
+          s.strings.loan_aave_fragment,
+          borrowEngineWallet.currencyInfo.currencyCode
```
Comment (samholmes) [inline src/controllers/action-queue/display.ts:113]: You could just change this to `toWallet.currencyInfo.currencyCode` and omit the op code change.

**Review by samholmes — CHANGES_REQUESTED** [review]

File: src/controllers/action-queue/cleaners.ts (hunk 1 above)
Comment (Jon-edge) [reply src/controllers/action-queue/cleaners.ts:116]: This is to support adding the AAVE network name to the swap description

File: src/controllers/action-queue/display.ts (hunk 2 above)
Comment (Jon-edge) [reply src/controllers/action-queue/display.ts:113]: I think it's fine to keep this explicit. I can think of a use case come up soon about wanting to swap for depositing, in which toWallet is no longer valid.

Comment (samholmes) [reply src/controllers/action-queue/display.ts:113]: `swap` action op is AAVE agnostic. Unless we want to rename it to `swap-aave` and implement the generic action op later (kinda like what we did with wyre-sell over fiat-sell).

**Review by samholmes — APPROVED** [review]

**Pull request merged**